api/tests/prosty.rs

```diff
@@ -0,0 +1,81 @@
+//! Tests that prosty versions of things in external.proto roundtrip with protobuf versions
```

Comment:
Opinion:

I find the word `prosty` (as in `prosty.rs`) to be confusing since it doesn't actually correspond to anything. Would the reader assume it's related to `prost` or would they google it assuming it might be something entirely different? I think we should stick to using the word `prost`.

(Although, while still a bit weird, using phrasing like `prost-y` in comments is at least not as confusing in the same way as `prosty` since it's more clear that it's referring to `prost`.)

Reply:
yeah if you don't like this I can change it to `prost.rs`

in casual conversation some engineers have referred to "prosty versions" of structs as opposed to the protobuf lib generated ones, that's where this came from

Reply:
done